docs/markdown/Builtin-options.md

```diff
@@ -209,6 +209,7 @@ available on all platforms or with all compilers:
 | b_staticpic         | true                 | true, false                                                   | Build static libraries as position independent                                 |
 | b_pie               | false                | true, false                                                   | Build position-independent executables (since 0.49.0)                          |
 | b_vscrt             | from_buildtype       | none, md, mdd, mt, mtd, from_buildtype, static_from_buildtype | VS runtime library to use (since 0.48.0) (static_from_buildtype since 0.56.0)  |
+| b_tasking_mil_link  | false                | true, false                                                   | Use MIL linking for the TASKING VX-tools compiler family (since 1.?.?)         |
```

Comment:
```suggestion
| b_tasking_mil_link  | false                | true, false                                                   | Use MIL linking for the TASKING VX-tools compiler family (since 1.4.0)         |
```